app/xeve_app.c

```diff
@@ -483,20 +483,22 @@ static int y4m_parse_tags(Y4M_INFO * y4m, char * tags)
 
 int y4m_header_parser(FILE * ip_y4m, Y4M_INFO * y4m)
 {
-    char buffer[80] = { 0 };
+    const size_t y4mheadersize = 256;
+
+    char buffer[y4mheadersize];
```

Comment:
Is it necessary though?

I deduced it my not be required since in `y4m_update_param` there's an [uninitialized buffer](https://github.com/mpeg5/xeve/blob/954ed6e0494cd2438fd15c717c0146e88e582b33/app/xeve_app.c#L542C5-L542C23).

Also the compiler wouldn't let me initialize it in the same static way since now the buffer is variable-sized
```c
xeve_app.c:488:34: error: variable-sized object may not be initialized except with an empty initializer
  488 |     char buffer[y4mheadersize] = { 0 };
  ```
If you prefer to initialize it to zeroes I can either use `memset` or initialize it like this:
```c
char buffer[y4mheadersize] = {};
```

Reply:
I think that memset will be the best option.